Gemfile

```diff
@@ -9,7 +9,7 @@ gem "inspec", path: "."
 # in it in order to package the executable. Hence the odd backwards dependency.
 gem "inspec-bin", path: "./inspec-bin"
 
-gem "ffi", ">= 1.9.14"
+gem "ffi", "1.12.2" # 1.13 does not work on Windows: https://github.com/ffi/ffi/issues/784
```

Comment:
Can we loosen this to [">= 1.9.14", "< 1.13"] to allow flexibility in resolving this when packaging?

Reply:
Good call, updated.